tests/pthScript

```diff
@@ -31,7 +31,7 @@ javac "-d java-out -cp ." {
 	java-src/ClassFile02.java;
 }
 
-polyglot.frontend.JLExtensionInfo "-assert -d out -cp java-out -postopts \"-source 1.6\"" {
+polyglot.frontend.JLExtensionInfo "-assert -d out -cp java-out" {
```

Comment:
Unfortunately, we have to remove the flag, otherwise the Java compiler will say something weird like `cannot find package java.lang in classpath and bootclasspath`. I suspect it's caused by the same issue of rt.jar and tools.jar being removed in Java 9+.